python/kserve/kserve/model_server.py

```diff
@@ -39,29 +39,46 @@
 DEFAULT_GRPC_PORT = 8081
 
 parser = argparse.ArgumentParser(add_help=False)
+# Model Server Arguments: The arguments are passed to the kserve.ModelServer object
 parser.add_argument("--http_port", default=DEFAULT_HTTP_PORT, type=int,
-                    help="The HTTP Port listened to by the model server.")
+                    help="The HTTP Port listened to by the model server. Default to 8080.")
```

Comment:
The defaults should already been generated in help doc if `default` is specified

Reply:
I do not see it in the doc, did I miss anything?

```
python3 -m huggingfaceserver --help
usage: __main__.py [-h] [--http_port HTTP_PORT] [--grpc_port GRPC_PORT] [--workers WORKERS] [--max_threads MAX_THREADS] [--max_asyncio_workers MAX_ASYNCIO_WORKERS] [--enable_grpc ENABLE_GRPC] [--enable_docs_url ENABLE_DOCS_URL]
                   [--enable_latency_logging ENABLE_LATENCY_LOGGING] [--configure_logging CONFIGURE_LOGGING] [--log_config_file LOG_CONFIG_FILE] [--access_log_format ACCESS_LOG_FORMAT] [--model_dir MODEL_DIR] [--model_id MODEL_ID]
                   [--enable_streaming ENABLE_STREAMING] [--tensor_parallel_degree TENSOR_PARALLEL_DEGREE] [--max_length MAX_LENGTH] [--do_lower_case DO_LOWER_CASE] [--add_special_tokens ADD_SPECIAL_TOKENS] [--task TASK]

options:
  -h, --help            show this help message and exit
  --http_port HTTP_PORT
                        The HTTP Port listened to by the model server.
  --grpc_port GRPC_PORT
                        The GRPC Port listened to by the model server.
  --workers WORKERS     The number of workers for multi-processing.
  --max_threads MAX_THREADS
                        The number of max processing threads in each worker.
  --max_asyncio_workers MAX_ASYNCIO_WORKERS
                        Max number of asyncio workers to spawn
  --enable_grpc ENABLE_GRPC
                        Enable gRPC for the model server
  --enable_docs_url ENABLE_DOCS_URL
                        Enable docs url '/docs' to display Swagger UI.
  --enable_latency_logging ENABLE_LATENCY_LOGGING
                        Output a log per request with latency metrics.
  --configure_logging CONFIGURE_LOGGING
                        Whether to configure KServe and Uvicorn logging
  --log_config_file LOG_CONFIG_FILE
                        File path containing UvicornServer's log config. Needs to be a yaml or json file.
  --access_log_format ACCESS_LOG_FORMAT
                        Format to set for the access log (provided by asgi-logger).
  --model_dir MODEL_DIR
                        A URI pointer to the model binary
  --model_id MODEL_ID   Huggingface model id
  --enable_streaming ENABLE_STREAMING
                        enable streaming mode
  --tensor_parallel_degree TENSOR_PARALLEL_DEGREE
                        tensor parallel degree
  --max_length MAX_LENGTH
                        max sequence length
  --do_lower_case DO_LOWER_CASE
                        do lower case
  --add_special_tokens ADD_SPECIAL_TOKENS
                        the sequences will be encoded with the special tokens relative to their model
  --task TASK           The task name
```

Reply:
Need to turn on the option for help formatting with default values.
https://docs.python.org/3/library/argparse.html#formatter-class

Reply:
Updated